src/pharmpy/tools/amd/run.py

```diff
@@ -246,6 +246,8 @@ def run_amd(
                 Transits((0, 1, 3, 10), Wildcard()),
                 Peripherals((0, 1, 2)),
             )
+        elif modeltype == 'tmdd':
+            modelsearch_features = (Elimination((Name('MM'), Name('MIX-FO-MM'))),)
```

Comment:
We still need absorption etc depending for oral drug and peripherals for iv or oral.